packages/docs/static/themes/cmsgov-theme.css

```diff
@@ -500,10 +500,4 @@
 --spacer-7: 56px;
 --spacer-none: 0px;
 --spacer-half: 4px;
---z-deep: -99999;
---z-default: 1;
---z-drawer: 500;
---z-dialog: 1000;
---z-popup: 6000;
---z-spinner: 9050;
```

Comment:
Just getting a head start on updating these files. This is really a meaningless change.